Source/CarthageKit/Git.swift

```diff
@@ -157,7 +157,7 @@ public func cloneRepository(cloneURL: GitURL, _ destinationURL: NSURL, bare: Boo
 		arguments.append("--bare")
 	}
 
-	return launchGitTask(arguments + [ "--quiet", cloneURL.URLString, destinationURL.path! ])
+	return launchGitTask(arguments + [ "--quiet", cloneURL.URLStringWithGithubAccessToken, destinationURL.path! ])
```

Comment:
It'd be nice to bake this in earlier if possible, so that `Git.swift` can know as little as possible about `GitHub.swift`.